### PR TITLE
Werewolf Rage Howl and Conversion Changes

### PR DESCRIPTION
**Pull request by GuppyLaxx**
ROUNDSTART WEREWOLVES ARE THE ONLY ONES WHO CAN CONVERT.

THEY GET 1 CONVERT PER NIGHT.

WEREWOLF STATS ARE HEAVILY NERFED, EXCEPT!!!!

YOU CAN NOW USE COMBAT MODE + HOWL TO COMBAT HOWL, CAUSING YOU TO GO INTO A RAGE THAT GIVES YOU STATS **BETTER THAN BEFORE THE NERFS.** AT THE COST OF YOUR VISION SLIGHTLY, AND ALERTING NEARBY PLAYERS.

### CODE REVIEW

**Comment by Heheniy** [comment]
dogshit

**Comment by X404ErrorX** [comment]
i think you should make every werewolf have atleast have one convertion or make the round start werewolf the only ones who can convert.

I think the werewolf spam is shit tbh but they should still have an way to convert people.

**Comment by john-pawson** [comment]
This would also mean that there would be a max of 4 werewolves per round... If you get lucky and get 2 werewolves that is, there's still a chance that only 1 werewolf spawns and get a max of 2 werewolves per round... wich would make werewolves more of a minor antag like the bandits rather than a major one, like vampyres or rebellions.
Also i'm pretty sure they're intended to be conversion antags rather than... whatever this makes them

**Comment by john-pawson** [comment]
Would you mind me asking what is it that you're exactly trying to fix here? As far as i know most people don't have much of a problem with werewolves being able to infect people as they can right now. Matter of fact the (currently) 7 thumbs down on the PR are telling me "Hey, we don't want this"

**Comment by TomGarou** [comment]
On the one hand, I understand the desire to focus werewolves more on one-on-one nightly conversions that involve a dedicated night for each convert for RP. 

However, it also has the major drawback of limiting accessibility to lycanthropy for other players. Few werewolves will risk converting the random alone in the bog unless he's known not to be a 'waste.' And that makes things less 'see prey and hunt/convert.' and more "What player is worth converting, and who do I kill quickly to get to that player." I don't want to see werewolves have to gamify like old vamps.

For an antag that cannot spawn during the middle of the round like bandits, this is a rare enough occurrence that many players look forward to.

Making it limited to only the elder werewolf for a faction with no hard allegiances also creates an even bigger investment in who you pick...since there's no guarantee they will be loyal to you. This is exasperated when you consider that a player who cannot convert only has two options—killing people who can't understand them and eating them, something that is likely going to be mechanical and rather droll for the one being eaten randomly by a wolf or the wolf becoming 'friendly' either becoming a merc wolf for vampires, local druids, or the crown. Or just chilling and ignoring the condition because killing isn't fun. It turns the player into a weapon with no clear goal and freedom to kill; when you all have a hammer, everything starts looking like a nail; unless you don't like hammering, then it all looks pointless.

Without a constructive outlet, 'Conversion,' there's not much point in being a werewolf besides killing with the language barriers

 From what I understand, the main issues with antagonist Conversion mechanics are twofold: 
 -> Immersion breaking via the forced control exasperated by the ever-ticking vitae time limit limiting your creativity and taking you out of your character's mindset to play undead nukies following your commander where only LRP gamer-ism can counter the ticking clock
 ->snowballing, creating chaotic environments where players get away with terrible things without consequence and a sense of helplessness. While a town of werewolves might be interesting now and again if folks roleplayed it out, unfortunately, it usually results in a mindset that conversion is no longer important or worth bothering with, leading to bloodbaths. (This is exactly what players default to when conversion is no longer feasible and other forms of interactions lead to hostilities.)
 
 , these are the things folks want to move away from, not the opportunity to have supernatural curses inflicted upon them that cause them to change how they interact with the round immersively.

Perhaps a more reasonable compromise would be one conversion per werewolf per night after your first full night for non-round starts, give the converts something to aspire to and survive for, and prevent snowballing. Also, it prevents the antag faction from being lynchpinned to the few lucky round starts. As it is, it would encourage werewolves to pack up together and overwhelm players to protect their valuable 'conversion' source instead of going off on their own, and encourage said source to only select those who he would know OOC would be loyal and effective to that endeavor


Werewolves need a greater purpose than just killing, and barring that...conversion is a positive interaction that adds value to both parties involved since it doesn't come with forced mechanical slavery and a complete requirement to abandon your character for whatever the daily vamp meta is. 

I think the very 'freeform' nature of werewolves makes them a desirable state for many players, and it's likely why you see so many thumbs-downs at the thought of less opportunity for that. 'what might my character do if he became such a horrible creature' is a fun question to ask. And unlike vampires, where the answer is always 'follow masters instructions, ' it might lead to an interesting outcome.

If you're trying to improve werewolf in the meantime before coming up with a more developed game mode. I think more guaranteed but slower conversions is reasonable, but do not cut pathways to conversions off entirely for players, it's the pure element of what makes werewolf most fun for both sides.

**Comment by radarseas** [comment]
what above said, if you remove it from ALL WEREWOLVES but starters, you suffer the issue that you're a liable roll for success of weres and thusly, you'd want to meta pick the best conversions. especally when well, people do what happened with vamp and immedately attack you upon converting despite going against team antag rules.


They should snowball if they've got decent numbers, they're supposed to be a threat and there should be reasoning to deal with them. However I will agree, giving the ability to have unlimited converts for everyone results in 2 day takeovers..

**Comment by DorfGuy1** [comment]
> On the one hand, I understand the desire to focus werewolves more on one-on-one nightly conversions that involve a dedicated night for each convert for RP.
> 
> However, it also has the major drawback of limiting accessibility to lycanthropy for other players. Few werewolves will risk converting the random alone in the bog unless he's known not to be a 'waste.' And that makes things less 'see prey and hunt/convert.' and more "What player is worth converting, and who do I kill quickly to get to that player." I don't want to see werewolves have to gamify like old vamps.
> 
> For an antag that cannot spawn during the middle of the round like bandits, this is a rare enough occurrence that many players look forward to.
> 
> Making it limited to only the elder werewolf for a faction with no hard allegiances also creates an even bigger investment in who you pick...since there's no guarantee they will be loyal to you. This is exasperated when you consider that a player who cannot convert only has two options—killing people who can't understand them and eating them, something that is likely going to be mechanical and rather droll for the one being eaten randomly by a wolf or the wolf becoming 'friendly' either becoming a merc wolf for vampires, local druids, or the crown. Or just chilling and ignoring the condition because killing isn't fun. It turns the player into a weapon with no clear goal and freedom to kill; when you all have a hammer, everything starts looking like a nail; unless you don't like hammering, then it all looks pointless.
> 
> Without a constructive outlet, 'Conversion,' there's not much point in being a werewolf besides killing with the language barriers
> 
> From what I understand, the main issues with antagonist Conversion mechanics are twofold: -> Immersion breaking via the forced control exasperated by the ever-ticking vitae time limit limiting your creativity and taking you out of your character's mindset to play undead nukies following your commander where only LRP gamer-ism can counter the ticking clock ->snowballing, creating chaotic environments where players get away with terrible things without consequence and a sense of helplessness. While a town of werewolves might be interesting now and again if folks roleplayed it out, unfortunately, it usually results in a mindset that conversion is no longer important or worth bothering with, leading to bloodbaths. (This is exactly what players default to when conversion is no longer feasible and other forms of interactions lead to hostilities.)
> 
> , these are the things folks want to move away from, not the opportunity to have supernatural curses inflicted upon them that cause them to change how they interact with the round immersively.
> 
> Perhaps a more reasonable compromise would be one conversion per werewolf per night after your first full night for non-round starts, give the converts something to aspire to and survive for, and prevent snowballing. Also, it prevents the antag faction from being lynchpinned to the few lucky round starts. As it is, it would encourage werewolves to pack up together and overwhelm players to protect their valuable 'conversion' source instead of going off on their own, and encourage said source to only select those who he would know OOC would be loyal and effective to that endeavor
> 
> Werewolves need a greater purpose than just killing, and barring that...conversion is a positive interaction that adds value to both parties involved since it doesn't come with forced mechanical slavery and a complete requirement to abandon your character for whatever the daily vamp meta is.
> 
> I think the very 'freeform' nature of werewolves makes them a desirable state for many players, and it's likely why you see so many thumbs-downs at the thought of less opportunity for that. 'what might my character do if he became such a horrible creature' is a fun question to ask. And unlike vampires, where the answer is always 'follow masters instructions, ' it might lead to an interesting outcome.
> 
> If you're trying to improve werewolf in the meantime before coming up with a more developed game mode. I think more guaranteed but slower conversions is reasonable, but do not cut pathways to conversions off entirely for players, it's the pure element of what makes werewolf most fun for both sides.

The thing about the chaotic states you mentioned,is that they can be fun, but at the same time, there's little time to RP, and I can guarantee that four out of five times someone is going to use it as an opportunity to do their best Greytide nation wide impression. We want fast enough to be interesting, but slow enough to have some time to RP things. Ultimately I think it's a player issue, as most of the time things go like the following. Brief spurt of essentials at round start, some establishing RP, more RP until something happens, and as soon as shit starts to get real rp is out the window. If there's a volf around then forget any rp, people are going to click that shit until horizontal and then keep clicking it again and again no matter what anyone says. People are locked in to a mindset where when they see that wolf, it's frag or be fragged. Any wolf that tries to roleplay is gonna get clicked until horizontal. Therefore it's up to the wolf to stick their neck out and try to break the cycle, but they're just gonna get sent to the respawn screen. We need to break the cycle somehow if we want good RP.

**Comment by Cyprex** [comment]
What is wrong with werewolves currently that this is needed? I think they're fine as-is

**Comment by UntoldTactics** [comment]
> What is wrong with werewolves currently that this is needed? I think they're fine as-is

Werewolves currently exist between two extremes.
A - Become friendly and don't convert many people. Tends to dull their influence, and overall makes them kind of a non-presence in a round.
B - Snowball out infecting more and more until the whole server is full of werewolves.

It has too much potential on both ends of the spectrum to simply ruin rounds or alternatively have no presence in rounds."
I've seen a lot of reasonings made here and there about werewolves but this somehow all fails to take into account the fact that werewolves have no imperative rush to them. It's possible to simply hide in 500 trillion spots in the bog where you'll never be found. There are hundreds of trees and each of them has a level 3 on the zlevel. Literally just climb a tree twice and unless the court magician scrys you, it's near impossible to find you. Is this kinda iffy? Sure, but on WW rounds the day lasts like eight minutes.

Conversion is hardly roleplayed out at all, and frankly roleplaying it within the confines of the setting and round-time aren't possible until precedents and lore is set. Too many have too many differing views on the function of werewolves in lore. Some might say they're mad entirely during their transformation but retain sanity as normal peeps, some say they'll be cognizant when a beast, etc. It's literally whatever the player themselves finds most appealing to their own rhetoric and it makes it impossible to consistently create conversion roleplay when said conversion has way too many expectations. Hell, there's always the 1/10 chance the convertee simply malds out at you, becomes friendly and starts hunting you down cursing your existence. That sounds fun on premise, but in reality it means you're literally just making more enemies for you to deal with, one who can listen in on you and has no consequence to doing so because they're not forced to actually fucking be hostile and no ones hostile to them in turn.

If people want to roleplay being horrible monsters, they shouldn't roleplay being one _whilst transformed_- the day should be used for the character to dimly realize the horror of their new existence. Their transformed state should be beastial and uncontrolled. If anything, wouldn't it make sense only the Elder Werewolfs have any control in their beast form? They've been werewolfs presumedly much longer. The Elders shouldn't be incharge because of any innate bullshit like VL obliterating your soul if you disobey, but by enforced roleplay precedent that only they have decent control over their beastial form and that the rest are driven by pack instinct and nudged around by the Elders to help their new lives.

I think people greatly underestimate how strong WW was, and has been. People say there's great risk in X and X but really the reality is the werewolfs have total freedom to engage and disengage at will. You do not have to stay fighting a risky fight and that alone makes them immensely powerful. You can take down practically any and all opponents in a 1v1, even a bloody inquisitor armed with silver and decked in full armor loses if you don't let them get a hit in on you (Rather easy when you can just tackle them from behind and slash their arm off whilst they can't even see you due to you being behind them)

Unlike say, VL, the werewolves remain at their full strength and always have that strength. There's 0 reason that people might be able to frag them early on because that's when EVERYONE is the least equipped to do so. The risk to the werewolf is simply only a risk if they choose to engage enemies they're not certain of besting. I genuinely believe a lot of early-round risks to werewolfs simply don't exist but are a product of players forgetting that retreat is always an option and that not fighting is sometimes the best way to fight. Genuinely, skill issue, get good here.

_TLDR;_
Werewolves **exist** to be cattle munching **monsters**. They exist to haunt the bog in small packs, to probe the town from advantage points only they can reach, to cause terror and occasional killings. They shouldn't be mass murdering, but neither should people expect mercy. You see a werewolf, it goes RAAAAAAGH with rage, then you fucking run for your life because you want to live. That's what a werewolf is, a fucking monster you're trying to escape. 

The 'roleplay' of being converted into a monster, only works if you are infact, **made into a monster.** What's the bloody point in conversion if half the time people act like friendly cuddly dogs? You can't make an excuse for roleplay in a time when the roleplay don't exist.

**Comment by SixplyDev** [comment]
> > What is wrong with werewolves currently that this is needed? I think they're fine as-is
> 
> Werewolves currently exist between two extremes. A - Become friendly and don't convert many people. Tends to dull their influence, and overall makes them kind of a non-presence in a round. B - Snowball out infecting more and more until the whole server is full of werewolves.
> 
> It has too much potential on both ends of the spectrum to simply ruin rounds or alternatively have no presence in rounds." I've seen a lot of reasonings made here and there about werewolves but this somehow all fails to take into account the fact that werewolves have no imperative rush to them. It's possible to simply hide in 500 trillion spots in the bog where you'll never be found. There are hundreds of trees and each of them has a level 3 on the zlevel. Literally just climb a tree twice and unless the court magician scrys you, it's near impossible to find you. Is this kinda iffy? Sure, but on WW rounds the day lasts like eight minutes.
> 
> Conversion is hardly roleplayed out at all, and frankly roleplaying it within the confines of the setting and round-time aren't possible until precedents and lore is set. Too many have too many differing views on the function of werewolves in lore. Some might say they're mad entirely during their transformation but retain sanity as normal peeps, some say they'll be cognizant when a beast, etc. It's literally whatever the player themselves finds most appealing to their own rhetoric and it makes it impossible to consistently create conversion roleplay when said conversion has way too many expectations. Hell, there's always the 1/10 chance the convertee simply malds out at you, becomes friendly and starts hunting you down cursing your existence. That sounds fun on premise, but in reality it means you're literally just making more enemies for you to deal with, one who can listen in on you and has no consequence to doing so because they're not forced to actually fucking be hostile and no ones hostile to them in turn.
> 
> If people want to roleplay being horrible monsters, they shouldn't roleplay being one _whilst transformed_- the day should be used for the character to dimly realize the horror of their new existence. Their transformed state should be beastial and uncontrolled. If anything, wouldn't it make sense only the Elder Werewolfs have any control in their beast form? They've been werewolfs presumedly much longer. The Elders shouldn't be incharge because of any innate bullshit like VL obliterating your soul if you disobey, but by enforced roleplay precedent that only they have decent control over their beastial form and that the rest are driven by pack instinct and nudged around by the Elders to help their new lives.
> 
> I think people greatly underestimate how strong WW was, and has been. People say there's great risk in X and X but really the reality is the werewolfs have total freedom to engage and disengage at will. You do not have to stay fighting a risky fight and that alone makes them immensely powerful. You can take down practically any and all opponents in a 1v1, even a bloody inquisitor armed with silver and decked in full armor loses if you don't let them get a hit in on you (Rather easy when you can just tackle them from behind and slash their arm off whilst they can't even see you due to you being behind them)
> 
> Unlike say, VL, the werewolves remain at their full strength and always have that strength. There's 0 reason that people might be able to frag them early on because that's when EVERYONE is the least equipped to do so. The risk to the werewolf is simply only a risk if they choose to engage enemies they're not certain of besting. I genuinely believe a lot of early-round risks to werewolfs simply don't exist but are a product of players forgetting that retreat is always an option and that not fighting is sometimes the best way to fight. Genuinely, skill issue, get good here.
> 
> _TLDR;_ Werewolves **exist** to be cattle munching **monsters**. They exist to haunt the bog in small packs, to probe the town from advantage points only they can reach, to cause terror and occasional killings. They shouldn't be mass murdering, but neither should people expect mercy. You see a werewolf, it goes RAAAAAAGH with rage, then you fucking run for your life because you want to live. That's what a werewolf is, a fucking monster you're trying to escape.
> 
> The 'roleplay' of being converted into a monster, only works if you are infact, **made into a monster.** What's the bloody point in conversion if half the time people act like friendly cuddly dogs? You can't make an excuse for roleplay in a time when the roleplay don't exist.

Snowballing or acting friendly is a player issue much more than it is a WW issue. WWs were fine as is and this PR will only kill the antag.

**Comment by HissyNays** [comment]
I'm all for verevolf nerfs but this honestly feels a bit heavy handed on stats.
That being said the infection changes are GREAT. 
Because verevolf spam every other round has been absolutely coal. It's an antag that was originally created to fight vampires and be limited so it's super strong. Now instead it was repurporsed to fight against town and somewhy gotten huge infection buff in past.

**Comment by Jackofashepard** [comment]
Cool aside from stats. Speed especially, giving them the option to run the fuck away is probably a better idea

**Comment by sirelgate** [comment]
How are you supposed to fight as a werewolf with nerfed stats lol. I get that you want to do a combat howl.

**Comment by Genevesh** [comment]
Werewolves have always been a bit of a glass canon, and taking away their speed even with the infinite stamina persisting feels like a bad choice.

The rage howl also makes little sense to me; Aren't they supposed to always be enraged anyway? Did you intend for it to be something of an indicator to start combat?

**Comment by HissyNays** [comment]
> Werewolves have always been a bit of a glass canon, and taking away their speed even with the infinite stamina persisting feels like a bad choice.
> 
> The rage howl also makes little sense to me; Aren't they supposed to always be enraged anyway? Did you intend for it to be something of an indicator to start combat?

Glass cannons don't step into bear traps and then continue chasing you while deflecting and dodging your spear stabs.
Verevolfs are incredibly resillient, incredibly fast, incredibly strong. Their only weakness is the player playing  them making mistakes. Or at least that was so before nerf, haven't encountered them with this PR.

> How are you supposed to fight as a werewolf with nerfed stats lol. I get that you want to do a combat howl.

It's still infinite stamina vibro blade meat grinding machine. I assume with this PR you just can't run balls deep into throne room and go whacking because 35 seconds won't be enough for whole fight.

**Comment by Koteshkin** [comment]
First you take out one of WW goals which is Vampire Lord.
Second you take away the second goal which is converts.

Now WW have NOTHING else to do, but either mindlessly kill people or stay AFK/ERP in the bog. 
clap

**Comment by Genevesh** [comment]
> > Werewolves have always been a bit of a glass canon, and taking away their speed even with the infinite stamina persisting feels like a bad choice.
> > The rage howl also makes little sense to me; Aren't they supposed to always be enraged anyway? Did you intend for it to be something of an indicator to start combat?
> 
> Glass cannons don't step into bear traps and then continue chasing you while deflecting and dodging your spear stabs. Verevolfs are incredibly resillient, incredibly fast, incredibly strong. Their only weakness is the player playing them making mistakes. Or at least that was so before nerf, haven't encountered them with this PR.
> 
> > How are you supposed to fight as a werewolf with nerfed stats lol. I get that you want to do a combat howl.
> 
> It's still infinite stamina vibro blade meat grinding machine. I assume with this PR you just can't run balls deep into throne room and go whacking because 35 seconds won't be enough for whole fight.

The issue is that they have some 'phantom' armor, which breaks in one or two hits; They are as vulnerable to leg crits as anyone else, and being crippled basically means death.

I've seen both skilled and not werewolves die to lucky crits in 1v1, let alone 1vMany scenarios; Just because your cheesy 'drop a beartrap at their feet' strategy doesn't work doesn't mean they're not easy to kill. I remind you that damage slows you down, especially damage to the legs, and getting hit has a staggering effect when you slow to the pace of a snail.

Werewolves are stripped of their only advantage in speed as soon as they get hit *once*, at which point they can easily get mobbed, and take a gigantic risk fighting anything more than one person at a time without backup. Their charges are easy as piss to dodge in the open area; And let's not mention them being completely defenseless against arrows.

Overall this PR discourages biting, nerfs them to only being able to fight during the rage timer, and encourages them to attack and more often than not frag loners that don't have a fighting chance against them, like the very isolated and easy to get to soilers.

**Comment by HissyNays** [comment]
>The issue is that they have some 'phantom' armor, which breaks in one or two hits; They are as vulnerable to leg crits as anyone else, and being crippled basically means death.

I've seen both skilled and not werewolves die to lucky crits in 1v1, let alone 1vMany scenarios; Just because your cheesy 'drop a beartrap at their feet' strategy doesn't work doesn't mean they're not easy to kill. I remind you that damage slows you down, especially damage to the legs, and getting hit has a staggering effect when you slow to the pace of a snail.

Werewolves are stripped of their only advantage in speed as soon as they get hit once, at which point they can easily get mobbed, and take a gigantic risk fighting anything more than one person at a time without backup. Their charges are easy as piss to dodge in the open area; And let's not mention them being completely defenseless against arrows.

Overall this PR discourages biting, nerfs them to only being able to fight during the rage timer, and encourages them to attack and more often than not frag loners that don't have a fighting chance against them, like the very isolated and easy to get to soilers.


The fact you said bear traps are cheesy says to me you're a big fan of playing as verevolf and bear traps just happen to be one of best way to slowdown a WW.
Pretty much everything you said there is just wrong. Verevolfes are absolutely limitless in potential. They can do a whole lot of things that they shouldn't be able to do. Even a verevolf downed with leg hits is absolutely deadly unless you can safely kite it with spear or bow. I've seen one fight 3 adventurers after being downed by them, even so he was downed because he was in fight with me. He lopped off legs of two and sent third one running while dragging one of his buddies. They can straight up lop off your limb through plate armor on first hit, i've had it happen.

**Comment by Genevesh** [comment]
> > The issue is that they have some 'phantom' armor, which breaks in one or two hits; They are as vulnerable to leg crits as anyone else, and being crippled basically means death.
> 
> > I've seen both skilled and not werewolves die to lucky crits in 1v1, let alone 1vMany scenarios; Just because your cheesy 'drop a beartrap at their feet' strategy doesn't work doesn't mean they're not easy to kill. I remind you that damage slows you down, especially damage to the legs, and getting hit has a staggering effect when you slow to the pace of a snail.
> 
> > Werewolves are stripped of their only advantage in speed as soon as they get hit once, at which point they can easily get mobbed, and take a gigantic risk fighting anything more than one person at a time without backup. Their charges are easy as piss to dodge in the open area; And let's not mention them being completely defenseless against arrows.
> 
> > Overall this PR discourages biting, nerfs them to only being able to fight during the rage timer, and encourages them to attack and more often than not frag loners that don't have a fighting chance against them, like the very isolated and easy to get to soilers.
> 
> The fact you said bear traps are cheesy says to me you're a big fan of playing as verevolf and bear traps just happen to be one of best ways to slowdown a WW. Pretty much everything you said there is just wrong. Verevolfes are absolutely limitless in potential. They can do a whole lot of things that they shouldn't be able to do. Even a verevolf downed with leg hits is absolutely deadly unless you can safely kite it with spear or bow. I've seen one fight 3 adventurers after being downed by them, even so he was downed because he was in fight with me. He lopped off legs of two and sent third one running while dragging one of his buddies. They can straight up lop off your limb through plate armor on first hit, i've had it happen.

I call it cheesy because it shouldn't be a feature; Reiterating you, the fact that you jump to conclusions says to me you're abusing shit like that for your advantage in combat and most likely just salting over a decently balanced antag. I've had it happen to me as a non-werewolf, getting typebaited, and then dragged into a trap they deploy from *their fucking satchel*, which automatically wins them a fight more often than not, how the hell does that make sense?

As for the whole lopping limbs off complaint, werewolves are pretty easily parried because their combat skill is worse than that of a watchman most of the time, and on defense intent you stop caring about them attacking rapidly; A downed werewolf is probably as dangerous as a leg crippled knight, that meaning they can and will murder you if you aren't being careful while finishing them off.

I've personally crippled werewolves with spears, flails, and swords - sure I haven't used all weapons against them but that basically covers most. I've SEEN people kite werewolves to death with bows, and in other cases kill them in a few shots with poisoned arrows - they are pushovers really; Meant to hit and run until they gather a big enough pack to cover each other in combat which is when they can actually contest with the less solitary combatants of the round.

**Comment by radarseas** [comment]
if you're GOING to insist on nerfing to 1 conversion and only elders, please, please, please, PLEASE, up the infection rates to something feasably deadly.

Currently unless you're brutally mangling and arterial biting people and leaving them in the hands of the church to bring back, werewolves absolutely suck at conversion odds and chances are your victim will be tossed to the graveyard because they _might-be-infected._

**Comment by JGlitch** [comment]
Why is this still merged when it has 31 negative votes and only 3 positive ones...?
